delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The SQL query directly interpolates user input, which is a security risk.

Recommendations:
- **Use Prepared Statements**: Modify the `DELETE` query to use prepared statements to prevent SQL injection.
- **Example**:
  ```php
  $stmt = $connection->prepare("DELETE FROM books WHERE id = ?");
  $stmt->bind_param("i", $id);
  $stmt->execute();
  ```